resize-event/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="A resize event demo on MDN" />
+    <title>Window Size Logger</title>
+  </head>
+  <body>
+    <p>Resize the browser window to fire the <code>resize</code> event.</p>
+    <p>Window height: <span id="height"></span></p>
+    <p>Window width: <span id="width"></span></p>
+
+    <script>
+      const heightOutput = document.querySelector("#height");
+      const widthOutput = document.querySelector("#width");
+
+      function resize() {
+        heightOutput.textContent = window.innerHeight;
+        widthOutput.textContent = window.innerWidth;
+      }
+
+      window.onresize = resize;
```

Comment:
I'd prefer us to use `addEventListener` here as we tend to recommend that method (https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#what_mechanism_should_i_use, https://developer.mozilla.org/en-US/docs/Web/API/eventtarget/addeventlistener).

Reply:
actually maybe that doesn't make sense since this is for the `onresize` property!

Reply:
Yes, but there's the same example on https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event page with a description :

> The following example reports the window size each time it is resized. Bear in mind that since the example is running in an <iframe>, you'll need to actually get the <iframe> to resize before you see an effect.

And it also shows how to add event listener using `addEventListener` - https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event#addeventlistener_equivalent

Reply:
Hey @AnilSeervi , and sorry to be so slow to look at this. I think this example should use `addEventListener`.

Quite soon we will merge the `onresize` and `resize_event` pages together, and then we will document `addEventListener` as the preferred form.

For examples embedded in the page, it makes sense to use the correct form for the page - so I mean, the `onresize` page should use `onresize`, and we'll take care of any updates when merging the pages. But because this is going to be a link from `onresize`, it'll be a pain to have to update it over here when we change the pages in mdn/content. So it's better if we make this one use `addEventListener` from the start.

Does that make sense?